python/glow/gwas/tests/test_lin_reg.py

```diff
@@ -19,7 +19,7 @@ def run_linear_regression(genotype_df, phenotype_df, covariate_df, fit_intercept
     Y[~Y_mask] = 0
     Q = np.linalg.qr(C)[0]
     Y = lr._residualize_in_place(Y, Q)
-    Y_state = lr._create_YState(Y, phenotype_df, None, Y_mask, np.float64)
+    Y_state = lr._create_YState(Y, phenotype_df, pd.DataFrame({}), Y_mask, np.float64)
```

Comment:
Why did we change this?

Reply:
With the new changes it was easier to take an empty dataframe since that matches the overall `linear_regression` function. It's private, so not a big deal to change.